src/deploy/osp_deployer/settings_sanity.py

```diff
@@ -46,6 +46,13 @@ def is_valid_ip(address):
 
         return valid
 
+    @staticmethod
+    def is_valid_mtu(mtu_val):
+        valid = False
+        if int(mtu_val) >= 594 and int(mtu_val) <= 9216:
+            valid = True
+        return valid
+
     def check_files(self):
```

Comment:
if the range is already enforced in the profiles json, you don't need the above.